package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "task2",
+  "version": "1.0.0",
+  "description": "webpack task",
+  "main": "index.js",
+  "scripts": {        
+    "start": "npm run webpack",
+    "webpack": "webpack -d --watch"
+},
+  "author": "",
+  "license": "ISC",
```

Comment:
You don't use js-beautify and pre-commit

Reply:
++